html5/js/Client.js

```diff
@@ -1546,9 +1542,9 @@ class XpraClient {
         "below",
         "title",
         "size-hints",
-        "class-instance",
+        CLASS_INSTANCE,
         "transient-for",
-        "window-type",
+        WINDOW_TYPE,
```

Comment:
Hmm, indeed. We could
1. accept this.
1. const-ify more strings to match.
1. increase this rule's tolerance to a minimum of 4 duplicate strings before it triggers (since these two strings are used 3 times each).
1. disable this rule for these specific strings (after enabling per-line rule disablement).
1. disable this rule for this specific file (after enabling per-file rule disablement).
1. abandon this rule and revert any questionable const-ified strings.

Reply:
It feels like cheating, but I quite like `3.`!
Then `2.`.
`4.` and `5.` feel too specific / complicated to configure, and the rules for the file would diverge from the rest - which I don't like on principle.
`6.` would be a shame, `1.` would be OK.

Reply:
Haha I agree 100%! Will do.

Reply:
Solved by https://github.com/Xpra-org/xpra-html5/pull/189/commits/772a8f44ae44e9152bb7db410a7638513342691b